NEWS.md

```diff
@@ -4,6 +4,9 @@
 
 ## Breaking changes
 
+* `str_trunc()` now correctly truncates strings when `side` is `"left"` or 
```

Comment:
Can you please put this in the news for the development version?

Reply:
Sorry for the mistake. I just fixed it.